### ISSUE TITLE
FR: Add flag to integrate my JSON-to-md Script

### ISSUE THREAD

**Issue by chrisgrieser** [mgmeyers/pdf-annots2json#7]
Reference: https://discord.com/channels/686053708261228577/928964558721265694/962298899714953227

The Script:
- https://github.com/chrisgrieser/pdf-annotation-extractor-alfred/blob/main/scripts/clean_annotations.js
- You are probably gonna need to also read what the annotations code do to understand the second half of the script: https://github.com/chrisgrieser/pdf-annotation-extractor-alfred#annotation-codes

**Comment by argenos** [comment]
Another vote for this!

**Comment by mgmeyers** [comment]
@chrisgrieser could you describe a bit more what you're imagining here?

Are you imagining a `--markdown` flag that will output markdown formatted in the same way as your script?

**Comment by chrisgrieser** [comment]
yeah, something like that would be one approach. (or rather `--markdown=suboption1,suboption2`, I guess?)

**Comment by tim-hilde** [comment]
Any updates regarding this? This is basically the only reason I'm not switching from the bibnotes formatter plugin

**Comment by mgmeyers** [comment]
@tim-hilde, could you say more? This FR is likely outside of the scope of this program, but I'm keeping it open in case I come across a good way to implement it.

What specifically is holding you back from switching from the bibnotes formatter plugin?
